storage/engine/mvcc.go

```diff
@@ -141,8 +143,13 @@ func (ms *MVCCStats) updateStatsOnPut(key proto.Key, origMetaKeySize, origMetaVa
 		// its contribution from live bytes in anticipation of adding in
 		// contribution from new version below.
 		if !orig.Deleted {
-			ms.LiveBytes -= (orig.KeyBytes + orig.ValBytes + origMetaKeySize + origMetaValSize)
+			ms.LiveBytes -= orig.KeyBytes + orig.ValBytes + origMetaKeySize + origMetaValSize
 			ms.LiveCount--
+			// Also, add the bytes from overwritten value to the GC'able bytes age stat.
+			ms.GCBytesAge += (orig.KeyBytes + orig.ValBytes) * origAgeSeconds
```

Comment:
I would encapsulate the multiplication here in a function since I suspect we may want to tweak the logic (do we really want to postpone GC of a gigabyte that was just deleted because another range has a megabyte that is an hour older?)